example_code/shmem_sync_example.c

```diff
@@ -5,43 +5,48 @@ int main(void)
 {
    static int x = 10101;
 
-   shmem_team_t         twos_team, threes_team;
+   shmem_team_t         twos_team   = SHMEMX_TEAM_INVALID;
+   shmem_team_t         threes_team = SHMEMX_TEAM_INVALID;
    shmem_team_config_t *config;
 
    shmem_init();
    config   = NULL;
    int me   = shmem_my_pe();
    int npes = shmem_n_pes();
 
-   int odd_npes = npes % 2;
+   if (npes > 2)
+       shmem_team_split_strided(SHMEMX_TEAM_WORLD, 2, 2, (npes-1) / 2, config,
+                                 0, &twos_team);
```

Comment:
Does the return from ```shmem_team_split_*``` guarantee a barrier on the parent team? Do we need a ```shmem_team_barrier(SHMEM_TEAM_WORLD)``` before the next team creation operation?

Reply:
No barrier needed.  :)